confgenerator/config.go

```diff
@@ -547,7 +547,7 @@ func (m *loggingProcessorMap) UnmarshalYAML(ctx context.Context, unmarshal func(
 }
 
 type LoggingService struct {
-	Compress  string               `yaml:"compress,omitempty" validate:"omitempty,oneof=gzip"`
+	Compress  string               `yaml:"compress,omitempty" validate:"omitempty,oneof=gzip,experimental=log_compression"`
```

Comment:
I think this will cause the confgenerator test to fail now, so you probably need to add this to the `init` in `confgenerator_test.go` (or regenerate the goldens if you want the unit test to just capture the error)